cves/kernel/CVE-2014-0196.yml

```diff
@@ -171,8 +169,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: an automated tool could have discovered this because beneath all of the concurrency race conditions it is basically a buffer overflow issue which can be discovered using a fuzzer to input many long strings to break the system
```

Comment:
Maybe describe what a fuzzer is. Someone reading this that has little knowledge of software security might not know what a fuzzer is.